src/api/app/models/event/repo_published.rb

```diff
@@ -3,7 +3,6 @@ class RepoPublished < Base
     self.message_bus_routing_key = 'repo.published'
     self.description = 'Repository was published'
     payload_keys :project, :repo, :buildid
-    after_save :clear_cache
 
     private
 
```

Comment:
@coolo, The next [private method](https://github.com/openSUSE/open-build-service/pull/6834/files#diff-80c5a72c12bac93b33994b23adcdd6d5R9) should be called `clear_caches` like the others?

Reply:
yep, will fix